### ISSUE TITLE
[Windows VS2019] RuntimeError when importing usd (extension class wrapper for base class ... has not been created yet)

### ISSUE THREAD

**Issue by firegreen** [PixarAnimationStudios/USD#1055]
### Description of Issue
I tried to compile USD with Visual Studio 2019 but when I run usdview.cmd or importing usd in python, the following error happens:
```python
Traceback (most recent call last):
  File "C:\software\USD\bin\usdcat", line 241, in <module>
    sys.exit(main())
  File "C:\software\USD\bin\usdcat", line 165, in main
    from pxr import Usd
  File "C:\software\USD\lib\python\pxr\Usd\__init__.py", line 24, in <module>
    import _usd
RuntimeError: extension class wrapper for base class struct pxrInternal_v0_19__pxrReserved__::Tf_PyEnumWrapper has not been created yet
```
Same USD works correctly when I compile with VS2015
### Steps to Reproduce
1. Build usd from master branch (commit v19.11 4b1162957b2ad219e1635c81de8343be490e41fc) with VS2019
2. Follow installation steps (set PATH and PYTHONPATH)
3. Import usd in python

### System Information (OS, Hardware)
Windows 10
Build with Visual Studio 2019 (VC 160)

### Build Flags
--generator "Visual Studio 16 2019" --build-args boost,"--with-date_time --with-thread --with-system --with-filesystem" --no-maya .\BUILD

### Notes
 * I tweak the build_usd.py for compiling Boost with VS 2019

**Comment by jilliene** [comment]
Filed as internal issue #USD-5745

**Comment by sunyab** [comment]
This is the same issue as #1095, which has been fixed in the the 20.05 release.
